codes/codes_test.go

```diff
@@ -91,3 +91,12 @@ func (s) TestUnmarshalJSON_MarshalUnmarshal(t *testing.T) {
 		}
 	}
 }
+
+func (s) TestUnmarshalJSON_InvalidIntegerCode(t *testing.T) {
+	var got Code
```

Comment:
Nit: move variable definition to be right before its first use

Reply:
Done